### DISCUSSION TITLE
Atlases for Resting State Atlas-based connectivity matrix?

### DISCUSSION THREAD

**Discussion by dastout** [HALFpipe/HALFpipe#546] [General] [answered]
Hello! We are looking to run atlas based resting-state connectivity. Are there atlases included in HALFpipe, and if so, how do I find them? In HALFpipe, I choose: [Atlas-based connectivity matrix] -> [corrMatrix]->specify my images to use -> But I don't know how to locate any atlases within HALFpipe (e.g., Powers, Freesurfer, etc.). Do I need to download these separately?

Thank you,

Danny

**Comment by HippocampusGirl** [comment, marked as answer]
Hi Danny,

We have prepared some atlases at https://github.com/HALFpipe/Atlases/releases/download/1.0.0/halfpipe-atlases.zip that you can download, extract the ZIP, and then choose one or more in the user interface. 

If you have your own atlases, then you will need to note down where they are stored on your disk, and navigate to those files in the user interface. 

Let me know if you're having any difficulties with that :-)

Best,
Lea

**Comment by dastout** [comment]
Hi Lea,

Great! Thank you!

-Danny
________________________________
